src/clr-addons/slider/slider.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="slider">
```

Comment:
this is even more of an wrapper and not the slider itself - class name should represent that, e.g. slider-outer-wrapper